lib/client/constructor.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Create instance of a Client
+ * @class
+ * @param {string} client_id Your Veryfi client id
+ * @param {string | undefined} client_secret Your Veryfi client secret
+ * @param {string} username Your Veryfi username
+ * @param {string} api_key Your Veryfi API key
+ * @param {string} base_url
+ * @param {string} api_version
+ * @param {Number} timeout
+ */
+function Client(
+    client_id,
+    client_secret,
+    username,
+    api_key,
+    base_url = "https://api.veryfi.com/",
+    api_version = "v8",
```

Comment:
let's not allow this to be changed, less chances somebody passes v7

Reply:
Done